src/spi.rs

```diff
@@ -705,6 +705,16 @@ pins! {
         MOSI: [PC1<Alternate<AF5>>]
 }
 
+/// Interrupt events
+pub enum Event {
+    /// New data has been received
+    Rxne ,
```

Comment:
```suggestion
    Rxne,
```

Please make sure to run `rustfmt` on your code.